spring-data-vmware-gemfire/src/test/java/org/springframework/data/gemfire/client/DurableClientCacheIntegrationTests.java

```diff
@@ -235,6 +236,7 @@ private void waitForRegionEntryEvents() {
 
     @Test
     @DirtiesContext
+		@Order(1)
```

Comment:
Is this indentation weird or is it just displaying that way in browser?

Reply:
I think it is just displaying weird. Looks "normal" in the IDE